src/style.css

```diff
@@ -0,0 +1,79 @@
+
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-weight: 300;
+}
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-weight: 900;
+}
+
+body {
+  margin-top: 20px;
+}
+
+.form {
+  position: relative;
+}
+
+.input {
+  width: 100%;
+  height: 70px;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px  #3d4e611a;
+  padding-left: 62px;
+  outline: #e1e7ed;
+  margin-bottom: 20px;
+  font-size: 16px;
+  color: #3d4e61;
+  font-family: Avenir, sans-serif;
+}
+
+.imput--small {
+  height: 42px;
+  font-size: 14px;
+  background-size: 11px 11px;
+  background-position: 15px 13px;
+  padding-left: 33px;
+}
+
+.input:hover {
+  background-color: lightgrey;
+  box-shadow: 0 3px 8px  #3d4e6133;
+  border: 1px solid blueviolet;
+}
+
+.input:focus {
+  border: 1px solid red;
+  background: linear-gradient(0.25turn, #3f87a6, #ebf8e1, #f69d3c);
+}
```

Comment:
https://user-images.githubusercontent.com/94289038/181201193-e0f51af5-c08c-4522-860d-bc8f40c93b0b.mov

looks like in sample, but your styles for hover and focus should be like in design!